systemtests/tests/py3plug-fd-postgresql/database/setup_local_db.sh

```diff
@@ -63,6 +63,7 @@ local_db_prepare_files() {
     echo "log_error_verbosity = verbose"
     echo "log_statement = 'all'"
     echo "log_temp_files = 0"
+    echo "checkpoint_timeout = 45"
```

Comment:
What does this do ?

Reply:
By default checkpoint are set to 5 minutes, but our tests timeout is lesser than that. Here the idea will be have PG forced to do something with the checkpoint during the tests live. But we can leave without it.